include/swift/Basic/ColorUtils.h

```diff
@@ -0,0 +1,45 @@
+//===--- ColorUtils.h - -----------------------------------------*- C++ -*-===//
+//
+// This source file is part of the Swift.org open source project
+//
+// Copyright (c) 2014 - 2016 Apple Inc. and the Swift project authors
+// Licensed under Apache License v2.0 with Runtime Library Exception
+//
+// See http://swift.org/LICENSE.txt for license information
+// See http://swift.org/CONTRIBUTORS.txt for the list of Swift project authors
+//
+//===----------------------------------------------------------------------===//
+//
+// This file defines an 'OSColor' class for helping printing colorful outputs
+// to the terminal.
+//
+//===----------------------------------------------------------------------===//
+
+#ifndef SWIFT_COLOR_UTILS_H
+#define SWIFT_COLOR_UTILS_H
```

Comment:
The macro guards style for all `swift/Basic` headers is to prefix with `SWIFT_BASIC_`, and follow with no underscores, so change to `SWIFT_BASIC_COLORUTILS_H`.